bundles/org.openhab.binding.saicismart/src/main/java/org/openhab/binding/saicismart/internal/SAICiSMARTBridgeHandler.java

```diff
@@ -188,7 +188,7 @@ public void initialize() {
                     logger.error("Could not get messages from SAIC iSMART account", e);
                 }
             }
-        }, 1, config.refreshInterval, TimeUnit.SECONDS);
+        }, 1, 10, TimeUnit.SECONDS);
```

Comment:
Please extract this hardcoded value and create a class (or binding) level constant